pwndbg/aglib/memory.py

```diff
@@ -32,6 +33,10 @@ def read(addr: int, count: int, partial: bool = False) -> bytearray:
         :class:`bytearray`: The memory at the specified address,
         or ``None``.
     """
+    if count > MAX_HEXDUMP_SIZE:
+        raise ValueError(
+            f"Request size too large: {count} bytes, max allowed is {MAX_HEXDUMP_SIZE} bytes"
+        )
```

Comment:
Please move this checks into `hexdump.py` commands:
- set `MAX_HEXDUMP_SIZE` to 5mb - please use print + error colour 
- add check for checkin `addr` is valid with - `if not pwndbg.aglib.memory.peek(addr)` - print error, invalid address